Yona/Yona/Others/SMSValidPasscodeLogin/Login/LoginViewController.swift

```diff
@@ -242,6 +186,22 @@ extension LoginViewController: CodeInputViewDelegate {
         })
     }
     
+    fileprivate func allowedNumberOfLoginAttemptsFailed() {
```

Comment:
Functions normally are named after what they do (e.g. ``registerFailedLoginAttempt``), with event handlers as the exception (e.g. ``onTooManyLoginAttempts``). In this particular case, the method name doesn't explain me the function of the function (pun intended).

Is this something like ``showMaxLoginAttemptsReachedScreen``?

Reply:
Yes, you are right. I've renamed to `showMaxLoginAttemptsReachedScreen`.